reporting/src/job-scheduler.js

```diff
@@ -783,11 +977,11 @@ export default class JobScheduler {
     return count;
   }
 
-  getTotalJobsInDlq() {
+  getTotalJobsWaitingForRetry() {
```

Comment:
nit (mostly because I cannot find anything in the PR :P)  `getTotalJobsWaitingForRetry = () => Object.values(this.jobQueues).reduce((count, q) => count + q.retryable?.length || 0, 0)`

Reply:
Not sure if it is more readable. I think, there is no consensus. Personally, I find loops more verbose, but easier to quickly grasp in such cases.